com.leinardi.gst.json

```diff
@@ -0,0 +1,158 @@
+{
```

Comment:
There are some unnecessary files around. At least `/include`, *.a` and `/lib/pkgconfig` should be added to `cleanup` array.

Reply:
Hey sorry, I don't get what you are referring to: which files are unnecessary? What and where should I add exaclty?

Reply:
I mean something similar to this: https://github.com/flathub/org.flatpak.Builder/blob/master/org.flatpak.Builder.json#L25

Reply:
Oh I see, how can I check if I need to clean something up? I opened a debug shell on the guest but I saw no `/include` nor `/lib/pkgconfig` folders on the root. Should I look somewhere else?

Anyway, I added the cleanup section to the manifest.